packages/client/src/components/plugins-panel.tsx

```diff
@@ -29,6 +39,26 @@ interface PluginsPanelProps {
   initialPlugins?: string[];
 }
 
+// Define a type for the essential plugin information
+type EssentialPluginInfo = {
+  title: string;
+  description: string;
+};
+
+// Map of essential plugins that require confirmation when removing
+const ESSENTIAL_PLUGINS: Record<string, EssentialPluginInfo> = {
+  '@elizaos/plugin-sql': {
+    title: 'Essential Plugin: SQL',
+    description:
```

Comment:
Can we update this to say something like you can remove it but you will need to replace it with an adapter plugin and ai plugin (please improve the English ) thanks please do this for both!